Userland/Libraries/LibJS/Bytecode/Interpreter.cpp

```diff
@@ -165,4 +165,35 @@ void Interpreter::continue_pending_unwind(Label const& resume_label)
         jump(resume_label);
     }
 }
+
+HashMap<UnderlyingType<Interpreter::OptimizationLevel>, NonnullOwnPtr<PassManager>> Interpreter::s_optimization_pipelines {};
```

Comment:
This intuitively feels like it should be an array instead of a hash map.

Reply:
Then the entries would all be allocated and initialised, or I'd have to switch to storing OwnPtr's 🤔 
I guess the second one works, though I'd have to add a dummy enum member for that.
Sure, why not